### DISCUSSION TITLE
How can I get static position with its variance information?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#303] [Q&A]
Hello everyone, I'd like to run PPP under static mode and get postition with corresponding covariace information, just like gamit. What should I do?
Hope someone to help and thanks a lot!

**Comment by tmayerguerr** [comment]
Hi,

Static positions can be estimated with [gnssParametrization->staticPositions](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#staticPositions).

Getting the covariances is a little more complicated as it is not directly implemented. You can write out the normal equation matrix additionally with [processingStep->writeNormalEquations](https://groops-devs.github.io/groops/html/gnssProcessingStepType.html#writeNormalEquations). This is the inverse of the covariance matrix. You should only select the relevant parameters with `remainingParameters->wildcard` with select type `'position.*'`

regards
Torsten

**Reply by FFFFFG-IGG** [reply]
> Hi,
> 
> Static positions can be estimated with [gnssParametrization->staticPositions](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#staticPositions).
> 
> Getting the covariances is a little more complicated as it is not directly implemented. You can write out the normal equation matrix additionally with [processingStep->writeNormalEquations](https://groops-devs.github.io/groops/html/gnssProcessingStepType.html#writeNormalEquations). This is the inverse of the covariance matrix. You should only select the relevant parameters with `remainingParameters->wildcard` with select type `'position.*'`
> 
> regards Torsten

Thanks Torsten！Now write out the normal equation is quite complicate and will take a lot disk space. I wonder if there is someway to output postition and its covariance in one file, just like the output format of gamit:
CoordX  COVX  CoordY COVY  CoordZ  COVZ
Maybe that could be finished by one program or set of programs in GROOPS? 
Regrads Fan
